packages/manager/modules/pci/src/projects/project/analytics-data-platform/deploy/deploy.html

```diff
@@ -0,0 +1,65 @@
+<section data-ng-init="localizedStorageUnit = ('analytics_data_platform_common_unit_gb' | translate)">
+    <!-- page header -->
+    <oui-page-header class="mb-2" data-heading="{{ ::'analytics_data_platform_deploy_title' | translate }}"></oui-page-header>
+    <!-- page content -->
+    <div>
+        <!-- error messages -->
+        <div class="mt-1 mt-sm-2" data-ng-if="$ctrl.messages.length > 0">
+            <oui-message
+                data-ng-repeat="message in $ctrl.messages track by $index"
+                data-ng-if="message.origin === 'analyticsDataPlatform.deploy'"
+                data-type="{{ message.type }}">
+                <span data-ng-bind="message.text"></span>
+            </oui-message>
+        </div>
+        <p data-translate="analytics_data_platform_deploy_description"></p>
+        <!-- deploy stepper -->
+        <div class="mt-1 mt-sm-2">
+            <oui-stepper>
+                <!-- general information -->
+                <oui-step-form data-header="{{ ::'analytics_data_platform_deploy_general_configuration_title' | translate }}"
+                               data-on-submit="$ctrl.submitGeneralInformation()">
+                    <div data-ng-include="'analytics-data-platform/deploy/general.html'"></div>
```

Comment:
This will take long time as deploy module is very complex and has lot of code written in controller. We don't have any intension or requirement of reusing any of these components. It will be specific to ADP deploy module. With this do you still see any reason to convert them to components?

Reply:
I would be better yes :)

Reply:
This has been done. All ng-includes have been replaced by components